containers/python-3-device-simulator-express/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,23 @@
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+#-------------------------------------------------------------------------------------------------------------
+
+# Update the VARIANT arg in devcontainer.json to pick a Python version: 3.7, 3.8 or higher
+ARG VARIANT=3.8
+FROM mcr.microsoft.com/vscode/devcontainers/python:${VARIANT}
+
+# [Optional] If your requirements rarely change, uncomment this section to add them to the image.
+#
+# COPY requirements.txt /tmp/pip-tmp/
+# RUN pip3 --disable-pip-version-check --no-cache-dir install -r /tmp/pip-tmp/requirements.txt \
+#    && rm -rf /tmp/pip-tmp
+
+# [Optional] Uncomment this section to install additional packages.
+#
+# RUN apt-get update \
+#     && export DEBIAN_FRONTEND=noninteractive \
+#    && apt-get -y install --no-install-recommends <your-package-list-here>
+
+# Create venv for device simulator
+RUN python3 -m venv /root/.vscode-server-insiders/extensions/ms-python.devicesimulatorexpress/venv
```

Comment:
You'll also need to do this for the non-root user (`vscode`) and `.vscode-server` for VS Code stable.  If you only think one venv is needed for all of them, you could create it in one spot and then give the `vscode` user access to it and add symlinks.

Reply:
ok let me check what I can do here.

Reply:
@Chuxel I have to take a closer look cause I noticed the venv folder also depends on the version vs code  in use: `.vscode-server-insiders`